File: Project_UI/angular-workspace/src/app/model/Tag.ts
///////////////////////////////////////////////////////////////////////////////////////////////////////
//  FILE : Templates.ts
//  AUTHOR : Pranav Sehgal <PranavSehgalCS>
//  DESCRIPTION: Is a ts model for tags with constructor to encapsulate data 
///////////////////////////////////////////////////////////////////////////////////////////////////////

export class Tag{
    public tagId:number = 0;
    public tagName:string = "";

    public constructor(tagId:number, tagName:string) {
        this.tagId = tagId;
        this.tagName = tagName;
    }
}

File: Project_API/src/main/java/com/library/model/Tag.java
package com.library.model;
///////////////////////////////////////////////////////////////////////////////////////////////////////    
//  FILE : Tag.java
//  AUTHOR : Pranav Sehgal <PranavSehgalCS>
//  DESCRIPTION: Is an tagmodel used to encapsulate data
///////////////////////////////////////////////////////////////////////////////////////////////////////

public class Tag {
    private int tagId;
    private String tagName;

    public Tag(int tagId, String tagName){
        this.tagId = tagId;
        this.tagName = tagName;
    }

    public int getTagId(){
        return tagId;
    }
    public String getTagName(){
        return tagName;
    }
}
